code/modules/projectiles/ammo_datums.dm

```diff
@@ -2856,6 +2856,27 @@
 /datum/ammo/bullet/shrapnel/jagged/on_hit_mob(mob/M, obj/item/projectile/P)
 	if(isXeno(M))
 		M.Slow(0.4)
+
+/*
+//========
+					CAS 30mm impacters
+//========
+*/
+/datum/ammo/bullet/shrapnel/gau  //for the GAU to have a impact bullet instead of firecrackers
+	name = "30mm Multi-Purpose shell"
+
+	damage = 150 //More damaging, but 2x less shells and low AP
```

Comment:
Drop down to 115, will most likely still be quite powerful but not an instant destroyer of all.

Reply:
done